includes/admin/class-wc-admin-post-types.php

```diff
@@ -892,9 +892,10 @@ public function woocommerce_media_upload_downloadable_product() {
 	 * @param int   $product_id product identifier.
 	 * @param int   $variation_id optional product variation identifier.
 	 * @param array $downloadable_files newly set files.
-	 * @deprecated and moved to post-data class.
+	 * @deprecated 3.3.0 and moved to post-data class.
 	 */
 	public function process_product_file_download_paths( $product_id, $variation_id, $downloadable_files ) {
+		wc_deprecated_function( 'WC_Admin_Post_Types::process_product_file_download_paths', '3.3', 'WC_Post_Data::process_product_file_download_paths' );
```

Comment:
There's a lot of confuse on it, since `WC_Post_Data::process_product_file_download_paths` is also deprecated:

https://github.com/woocommerce/woocommerce/blob/86f59dff09529c52beead45e01e4b59f014c6040/includes/class-wc-post-data.php#L508-L510

Reply:
@claudiosanches Then this should have no replacement as well, agreed?